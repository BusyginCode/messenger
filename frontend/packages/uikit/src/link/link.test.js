import React from 'react'
import { render } from 'enzyme'
import Link from './link'

describe('uikit/Link', () => {
  it('should render correctly', () => {
    const props = {
      intent: 'primary',
      disabled: true,
      underlined: true,
      className: 'className',
      href: '/'
    }
    expect(render(<Link {...props}>link</Link>)).toMatchSnapshot()
  })
})
