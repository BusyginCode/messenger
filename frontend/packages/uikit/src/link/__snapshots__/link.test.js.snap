// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Link should render correctly 1`] = `
<h2
  class="className link primary disabled underlined base general m left"
>
  link
</h2>
`;
